lib/kafo/version.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: UTF-8
 module Kafo
   PARSER_CACHE_VERSION = 1
-  VERSION = "4.1.0"
+  VERSION = "4.2.0"
```

Comment:
Given https://github.com/theforeman/kafo/commit/68ae1ad760f4073a6fedc9ce8cfe4f6ddd1a85e8 and https://github.com/theforeman/kafo/commit/585c108d61f42851c14cbe4ccc4086638fad74bd it feels like a 5.0.0 release.